python/sdk/merlin/docker/docker.py

```diff
@@ -54,3 +56,21 @@ def _copy_dockerfile(dst, dockerfile_name):
     full_path = os.path.join(package_path, dockerfile_name)
     shutil.copy(full_path, dst_dockerfile_path)
     return dst_dockerfile_path
+
+
+def wait_build_complete(logs, debug: bool = False):
```

Comment:
just curious, does this function use to stream the logs or just print the logs after the build is finished? @ariefrahmansyah

Reply:
Great question. This function streams the docker build logs instead of waiting for the build to finish first. So the logs are streamed in real-time (if `debug` argument is `True`).

Reply:
you may resolved this issue